schema.sql

```diff
@@ -9,3 +9,27 @@ CREATE TABLE animals (
     weight_kg DECIMAL,
     species varchar(100)
 );
+
+-- owners table 
+CREATE TABLE owners (
+    id INT,
+    full_name varchar(100),
+    age INT,
+    PRIMARY KEY(owner_id)
+);
+
+-- species table 
+CREATE TABLE species (
+    id INT,
+    name varchar(100),
+    PRIMARY KEY(species_id)
+);
```

Comment:
[OPTIONAL]

- Primary keys are meant to be columns that uniquely identify a row. Foreign key links two tables together, so it is highly possible that different rows would have the same foreign key value, hence making it unfit to be the primary key. Please, go through this StackOverflow [discussion](https://stackoverflow.com/questions/10982992/is-it-fine-to-have-foreign-key-as-primary-key/10983016#:~:text=9%20Answers&text=Foreign%20keys%20are%20almost%20always,act%20as%20the%20primary%20key.) to get more information. 👍